pytrip/vdx.py

```diff
@@ -704,6 +705,10 @@ def __init__(self, name, cube=None):
         self.center_pos = None
         self.polygon3d = None
         self.voi_cube = None
+        self._slices_sagittal = []
+        self._slices_sagittal_range = None
```

Comment:
add comment about `self._slices_sagittal_range` and `self._slices_coronal_range` being used only for caching